Add references for four methods in the code not described in the book (#1216)
browserengineering/book: book/animations.md

animations that only require the draw step, like our example opacity
animation, will now run super smoothly.

> Final note: the compositing data structures need to be cleared when changing

Reviewer: ```suggestion
One final note: the compositing data structures need to be cleared when changing
```

> tabs, so let's do that (factoring out a `clear_data` method to do it more
> cleanly now that there are several things to clear):

Reviewer: ```suggestion
tabs. Let's do that by factoring out a `clear_data` method that clears everything in one go.
```
